### ISSUE TITLE
Case Sensitive Paths

### ISSUE THREAD

**Issue by yourmediafriend** [alvarotrigo/react-fullpage#12]
> ./node_modules/@fullpage/react-fullpage/dist/react-fullpage.js
Module not found: `/XXXX/site/node_modules/React/index.js` does not match the corresponding path on disk `react`.

my quick hack fix to `dist/react-fullpage.js`

```javascript
(function webpackUniversalModuleDefinition(root, factory) {
	if(typeof exports === 'object' && typeof module === 'object')
		module.exports = factory(require("react"));
	else if(typeof define === 'function' && define.amd)
		define("ReactFullpage", ["React"], factory);
	else if(typeof exports === 'object')
		exports["ReactFullpage"] = factory(require("react"));
	else
		root["ReactFullpage"] = factory(root["react"]);
})(window, function(__WEBPACK_EXTERNAL_MODULE__0__)
```

**Comment by danielimmke** [comment]
This has been fixed on dev see #9

**Comment by alvarotrigo** [comment]
@danielimmke thanks for pointing it out! I'll make sure to merge it on the master branch and publish it in npm as soon as possible!

**Comment by alvarotrigo** [comment]
Fixed now in fullpage-react 0.0.6
